packages/studio-base/src/components/SidebarContent.tsx

```diff
@@ -2,130 +2,113 @@
-  /** Buttons/items to display on the trailing (right) side of the header */
-  trailingItems?: React.ReactNode[];
-}>): JSX.Element {
+}: React.PropsWithChildren<SidebarContentProps>): JSX.Element {
   const classes = useStyles();
-  const theme = useTheme();
   const [showHelp, setShowHelp] = useState<boolean>(false);
-  const button = useTooltip({ contents: showHelp ? "Hide help" : "Show help" });
 
   const trailingItemsWithHelp = useMemo(() => {
     if (helpContent != undefined) {
       return [
         ...(trailingItems ?? []),
         <IconButton
-          elementRef={button.ref}
+          color={showHelp ? "inherit" : "primary"}
+          title={showHelp ? "Hide help" : "Show help"}
           key="help-icon"
-          iconProps={{ iconName: showHelp ? "HelpCircleFilled" : "HelpCircle" }}
           onClick={() => setShowHelp(!showHelp)}
-          styles={{
-            icon: {
-              color: theme.semanticColors.bodySubtext,
-
-              svg: {
-                fill: "currentColor",
-                height: "1em",
-                width: "1em",
-              },
-            },
-          }}
         >
-          {button.tooltip}
+          {showHelp ? <HelpIcon /> : <HelpOutlineIcon />}
         </IconButton>,
       ];
     }
     return trailingItems ?? [];
-  }, [helpContent, trailingItems, button, showHelp, theme]);
+  }, [helpContent, trailingItems, showHelp]);
 
   return (
-    <div className={classes.root}>
-      <div className={classes.toolbar}>
+    <Stack overflow={overflow} fullHeight flex="auto" gap={1}>
+      <Stack
+        className={classes.toolbar}
+        direction="row"
+        justifyContent="space-between"
+        alignItems="center"
+        padding={2}
+      >
         {leadingItems && (
-          <div className={classes.items}>
+          <Stack direction="row" alignItems="center">
             {leadingItems.map((item, i) => (
               <div key={i}>{item}</div>
             ))}
-          </div>
+          </Stack>
         )}
-        <Text as="h2" variant="xLarge" styles={{ root: { flexGrow: 1, margin: 0 } }}>
+        <Typography component="h2" variant="h4" fontWeight={800}>
           {title}
-        </Text>
+        </Typography>
         {trailingItemsWithHelp.length > 0 && (
-          <div className={classes.items}>
+          <Stack direction="row" alignItems="center">
             {trailingItemsWithHelp.map((item, i) => (
               <div key={i}>{item}</div>
             ))}
-          </div>
+          </Stack>
         )}
-      </div>
+      </Stack>
       {showHelp && (
         <div className={classes.helpContent}>
           <TextContent allowMarkdownHtml={true}>{helpContent}</TextContent>
         </div>
       )}
-      <div
-        className={cx(classes.content, {
-          [classes.noPadding]: noPadding,
-        })}
-      >
-        {children}
-      </div>
-    </div>
+      <ContentWrapper {...{ disablePadding }}>{children}</ContentWrapper>
```

Comment:
A little less duplication is not worth it here IMO
```suggestion
      <ContentWrapper disablePadding={disablePadding}>{children}</ContentWrapper>
```